src/app/AppLayout/AppLayout.tsx

```diff
@@ -237,10 +280,20 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
     </DropdownToggle>
   );
 
+  // TODO refactor to use Masthead, Toolbar components: https://www.patternfly.org/v4/components/page
   const HeaderTools = (
     <>
       <PageHeaderTools>
         <PageHeaderToolsGroup>
+          <FeatureFlag level={FeatureLevel.DEVELOPMENT}>
```

Comment:
Should this be using strict comparison too?

Reply:
It could, though it doesn't make a difference in this case.

Reply:
Ahh yeh I was thinking adding strict for clearer presentation :D